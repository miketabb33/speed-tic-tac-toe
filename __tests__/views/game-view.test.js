import React from 'react'
import renderer from 'react-test-renderer'
import GameView from '../../views/game-view.js'

test('render', () => {
	const subject = renderer.create(
		<GameView />
	)

	const result = subject.toJSON()
	expect(result).toMatchSnapshot()
})