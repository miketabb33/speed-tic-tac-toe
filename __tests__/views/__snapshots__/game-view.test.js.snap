// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<div>
  <div
    class="topBar"
  >
    <div
      className="timerView"
    >
      X 10:00
    </div>
    <div
      className="textDisplay"
    >
      X Goes First
    </div>
    <div
      className="timerView"
    >
      X 10:00
    </div>
  </div>
  <div
    className="timerInputView"
  >
    <div
      className="timerInputText"
    >
      Starting Time:
    </div>
    <input
      className="timerInput"
      maxLength="2"
      onKeyPress={[Function]}
      type="text"
    />
  </div>
  <div
    className="boards"
  >
    <div
      className="board"
    >
      <div
        className="boardImage"
      >
        <div
          style={
            Object {
              "boxSizing": "border-box",
              "display": "inline-block",
              "margin": 0,
              "maxWidth": "100%",
              "overflow": "hidden",
              "position": "relative",
            }
          }
        >
          <div
            style={
              Object {
                "boxSizing": "border-box",
                "display": "block",
                "maxWidth": "100%",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzMwIiBoZWlnaHQ9IjMzMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
              style={
                Object {
                  "border": "none",
                  "display": "block",
                  "margin": 0,
                  "maxWidth": "100%",
                  "padding": 0,
                }
              }
            />
          </div>
          <img
            data-nimg="intrinsic"
            decoding="async"
            draggable="false"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              data-nimg="intrinsic"
              decoding="async"
              draggable="false"
              loading="lazy"
              src="/_next/image?url=%2Fboard.png&w=750&q=75"
              srcSet="/_next/image?url=%2Fboard.png&w=384&q=75 1x, /_next/image?url=%2Fboard.png&w=750&q=75 2x"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </div>
      </div>
      <div
        className="boardSquares"
      >
        <div>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
        </div>
        <div>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
        </div>
        <div>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
        </div>
      </div>
      <div
        className="boardTitle"
      >
        Top
      </div>
    </div>
    <div
      className="board"
    >
      <div
        className="boardImage"
      >
        <div
          style={
            Object {
              "boxSizing": "border-box",
              "display": "inline-block",
              "margin": 0,
              "maxWidth": "100%",
              "overflow": "hidden",
              "position": "relative",
            }
          }
        >
          <div
            style={
              Object {
                "boxSizing": "border-box",
                "display": "block",
                "maxWidth": "100%",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzMwIiBoZWlnaHQ9IjMzMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
              style={
                Object {
                  "border": "none",
                  "display": "block",
                  "margin": 0,
                  "maxWidth": "100%",
                  "padding": 0,
                }
              }
            />
          </div>
          <img
            data-nimg="intrinsic"
            decoding="async"
            draggable="false"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              data-nimg="intrinsic"
              decoding="async"
              draggable="false"
              loading="lazy"
              src="/_next/image?url=%2Fboard.png&w=750&q=75"
              srcSet="/_next/image?url=%2Fboard.png&w=384&q=75 1x, /_next/image?url=%2Fboard.png&w=750&q=75 2x"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </div>
      </div>
      <div
        className="boardSquares"
      >
        <div>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
        </div>
        <div>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
        </div>
        <div>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
        </div>
      </div>
      <div
        className="boardTitle"
      >
        Middle
      </div>
    </div>
    <div
      className="board"
    >
      <div
        className="boardImage"
      >
        <div
          style={
            Object {
              "boxSizing": "border-box",
              "display": "inline-block",
              "margin": 0,
              "maxWidth": "100%",
              "overflow": "hidden",
              "position": "relative",
            }
          }
        >
          <div
            style={
              Object {
                "boxSizing": "border-box",
                "display": "block",
                "maxWidth": "100%",
              }
            }
          >
            <img
              alt=""
              aria-hidden={true}
              src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzMwIiBoZWlnaHQ9IjMzMCIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB2ZXJzaW9uPSIxLjEiLz4="
              style={
                Object {
                  "border": "none",
                  "display": "block",
                  "margin": 0,
                  "maxWidth": "100%",
                  "padding": 0,
                }
              }
            />
          </div>
          <img
            data-nimg="intrinsic"
            decoding="async"
            draggable="false"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style={
              Object {
                "border": "none",
                "bottom": 0,
                "boxSizing": "border-box",
                "display": "block",
                "height": 0,
                "left": 0,
                "margin": "auto",
                "maxHeight": "100%",
                "maxWidth": "100%",
                "minHeight": "100%",
                "minWidth": "100%",
                "objectFit": undefined,
                "objectPosition": undefined,
                "padding": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": 0,
              }
            }
          />
          <noscript>
            <img
              data-nimg="intrinsic"
              decoding="async"
              draggable="false"
              loading="lazy"
              src="/_next/image?url=%2Fboard.png&w=750&q=75"
              srcSet="/_next/image?url=%2Fboard.png&w=384&q=75 1x, /_next/image?url=%2Fboard.png&w=750&q=75 2x"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
          </noscript>
        </div>
      </div>
      <div
        className="boardSquares"
      >
        <div>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
        </div>
        <div>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
        </div>
        <div>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
          <button
            className="square"
            onClick={[Function]}
          >
            <svg
              height={79}
              width={79}
            >
              <rect
                className="clear"
                height={79}
                width={79}
              />
            </svg>
          </button>
        </div>
      </div>
      <div
        className="boardTitle"
      >
        Bottom
      </div>
    </div>
  </div>
</div>
`;
